Week 1/Weekly Practice Problems (Week 1)/0412: Fizz Buzz/Mohammad.cpp

```diff
@@ -0,0 +1,27 @@
+class Solution {
+public:
+    vector<string> fizzBuzz(int n) {
+        vector<string> v;
+        for(int i = 1 ; i <= n ; i++){
```

Comment:
To do for practice:
Use i for indexing and another variable to tell whether you've reached a clean division of 3 or 5.